octopoes/tools/rename-origin-method.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+import json
+import logging
+from typing import Any
+
+import click
+from xtdb_client import XTDBClient
+
+logger = logging.getLogger(__name__)
+
+
+@click.group(
+    context_settings={
+        "help_option_names": ["-h", "--help"],
+        "max_content_width": 120,
+        "show_default": True,
+    }
+)
+@click.option("-n", "--node", default="0", help="XTDB node")
```

Comment:
The `0` doesn't make much sense as default for the node. The node name is the organisation code which is what we use/ask in most of the other places, so I think it makes sense to also use that here.

The argument name also needs to be changed, but I can't do that in the suggestion.

```suggestion
@click.option("-c", "--code", default="0", help="The organisation code")
```